From 5c13ed4eeeb151b5bc648eb6f7262f65903470c4 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Fri, 7 Jun 2013 14:07:57 +0900
Subject: [PATCH 05/14] omxh264dec: allow to accept H.264 stream in a nal unit

This change allows to set caps to "alignment=nal" in order to
receive H.264 stream that is split into a nal unit.
---
 omx/gstomxh264dec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/omx/gstomxh264dec.c b/omx/gstomxh264dec.c
index 2581889..3ed41fc 100644
--- a/omx/gstomxh264dec.c
+++ b/omx/gstomxh264dec.c
@@ -61,7 +61,7 @@ gst_omx_h264_dec_class_init (GstOMXH264DecClass * klass)
 
   videodec_class->cdata.default_sink_template_caps = "video/x-h264, "
       "parsed=(boolean) true, "
-      "alignment=(string) au, "
+      "alignment=(string) { au, nal }, "
       "stream-format=(string) byte-stream, "
       "width=(int) [1,MAX], " "height=(int) [1,MAX]";
 
-- 
1.8.1.2

